widgetssdk/src/main/java/com/glia/widgets/chat/controller/ChatController.kt

```diff
@@ -1288,6 +1289,20 @@ internal class ChatController(
         )
     }
 
+    private fun markMessagesReadWithDelay() {
```

Comment:
Are we not clearing the disposable?

Reply:
But why do you think we need to do it here?